src/main/java/com/fasterxml/jackson/module/kotlin/SpreadWrapper.java

```diff
@@ -0,0 +1,46 @@
+package com.fasterxml.jackson.module.kotlin;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * Wrapper to avoid costly calls using spread operator.
+ */
```

Comment:
Read comments above and yes, keeping this against 2.12, changing only when otherwise done is fine!
Adding `@since 2.13` for new classes and methods is something I recommend, sometimes easier to read changes later on (I know git log has all info, but that requires looking for it).

Reply:
Added comment.
bc3581a

As for the change to the 2.13 branch, I'll do that next weekend.